edx_event_bus_redis/internal/producer.py

```diff
@@ -11,10 +11,12 @@
 from edx_django_utils.monitoring import record_exception
 from openedx_events.data import EventsMetadata
 from openedx_events.event_bus import EventBusProducer
+from openedx_events.event_bus.avro.tests.test_utilities import serialize_event_data_to_bytes
```

Comment:
This is waiting on the change here, correct? https://github.com/openedx/openedx-events/pull/189

Reply:
Looks like that change just merged, so I'll thumb this when this PR is updated to use that 👍

Reply:
@bmtcril Updated. :+1: